htdocs/core/class/html.form.class.php

```diff
@@ -233,7 +234,10 @@ public function editfieldval($text, $htmlname, $value, $object, $perm, $typeofda
 		if (!empty($conf->global->MAIN_USE_JQUERY_JEDITABLE) && !preg_match('/^select;|day|datepicker|dayhour|datehourpicker/', $typeofdata)) { // TODO add jquery timepicker and support select
 			$ret .= $this->editInPlace($object, $value, $htmlname, $perm, $typeofdata, $editvalue, $extObject, $custommsg);
 		} else {
-			$editmode = (GETPOST('action', 'aZ09') == 'edit'.$htmlname);
+			if ($editaction == '') {
```

Comment:
What is goal/use case of this framework enhancement ? Also because it is a framework enhancemt, can you push it on a separate PR ?

Reply:
@eldy 
I removed the enhancement part as you suggested to me.
I'm going to add a separate PR for editfieldval() method.

Reply:
Separate this PR into PR #23825